fe/fe-core/src/main/java/com/starrocks/load/DeleteHandler.java

```diff
@@ -919,6 +923,10 @@ public void replayMultiDelete(MultiDeleteInfo deleteInfo, GlobalStateMgr globalS
         if (deleteInfo == null) {
             return;
         }
+        if (isDeleteInfoExpired(deleteInfo, System.currentTimeMillis())) {
```

Comment:
You should add this check after updateTableDeleteInfo(), because it will update some flag of its related table.

Reply:
Done